claude.md

```
@@ -1,434 +1,85 @@
-- `README.md` - Main project documentation
-- `CHANGELOG.md` - Project changelog
-- `AGENTS.md` - AI agent instructions
-- `CLAUDE.md` - This file (Claude-specific instructions)
-- `00_START_HERE.md` - Getting started guide (if applicable)
-
-### Documentation Structure
-
-All other `.md` files must be organized in `docs/` subdirectories:
+## Development
 
+### Python setup
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)